README.md

```diff
@@ -0,0 +1,54 @@
+# Instagram Clone
+
+Instagram clone built with React, Redux, Node.js, Express, Passport.js & MongoDB.
+
+## Features
+
+- Sign up for an account and edit avatar/bio/name later on
+- Upload images with captions and option to edit/delete them
+- Comment on posts and option to edit/delete them
+- Like/unlike posts
+- Follow/unfollow users
+- Feed consists of posts from users you follow and the 'Explore' page has posts from every user
+
+## Features to add
+
+- Demo login button (for people who don't want to register)
+- Add location to posts
+- Add tags to posts
+- Search option for tags and location
+- Bookmark posts
+- Modals that display likes (on a post), and followers/following (on profile)
+
+## Running Locally
+
+The following instructions should get this project running on your machine.  
+
+### Prerequisites
+1. Install [MongoDB](https://www.mongodb.com/)  
```

Comment:
You need not install mongo, you have to connect your mongo db with your project by creating a  free mongo db account and adding the database user name and databse password in the keys.js.example file